src/components/App/App.tsx

```diff
@@ -24,11 +55,42 @@ const App = (): JSX.Element => {
             <Route exact path="/">
               <Form updateUserSentiment={ updateUserSentiment } />
             </Route>
-            <Route path="/feed/">
-              <Feed userSentiment={ userSentiment } />
-            </Route>
+            <Route
+              exact path="/feed/"
+              render={() => {
+                return (
+                  <>
+                    <Feed
+                      userSentiment={ userSentiment }
+                      articles={ articles }/>
+                    { !articles.length && <h2>Loading.. </h2>}
```

Comment:
I added this loading conditional rendering here, in case the user clicks their sentiment really fast and the articles are available yet. I also added the error conditional rendering, but we can put whatever text we want inside of that `h2`